### PR TITLE
nette 2.4 compatility

### LINKED ISSUE #39: support for Nette 2.4
Hi, in nette 2.4 i have some problems. If I install by composer your package and register it into config.neon.

Nette show me blue screen (tracy) with this:
`Missing annotation @property for Brabijan\Images\DI\ImagesExtension::$containerBuilder used in /nette24/vendor/brabijan/images/src/Brabijan/Images/DI/ImagesExtension.php:69`

How can i fix this problem? Thanks

### PR DESCRIPTION
**Pull request by CZechBoY**
Closes #39
and merges #25

### CODE REVIEW

**Comment by brabijan** [comment]
thanks a lot

**Comment by chemix** [comment]
why src/Brabijan/Images/Forms/UploadControl.php is deleted?

**Comment by brabijan** [comment]
Because only reason, why it was in package was missing multiupload feature in nette forms. There's no need for that anymore

**Comment by chemix** [comment]
👍

File: src/Brabijan/Images/Macros/Latte.php [hunk 1]
```diff
@@ -132,11 +131,15 @@ public function finalize()
...
 
 	/**
-	 * @param \Nette\Templating\Template $template
+	 * @param Latte\Runtime\Template|Nette\Application\UI\Template $template
 	 * @throws \Nette\InvalidStateException
 	 */
-	public static function validateTemplateParams(Template $template)
+	public static function validateTemplateParams($template)
 	{
+        if (!($template instanceof \Latte\Runtime\Template) || !($template instanceof \Nette\Application\UI\Template)) {
```
Comment (TomasVotruba) [inline src/Brabijan/Images/Macros/Latte.php]: Fix indentation please

**Pull request merged**